<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cukrzyca 2.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(2, 36, 65);
color: rgb(246, 245, 244);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(19, 56, 91);
border:1px solid;
border-color: rgb(28, 113, 216);
border-radius:5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz datę pomiaru:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>16</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pokazWyniki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Carlito</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Okno pomiarów poziomu cukru, wagi, ciśnienia</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(19, 56, 91);
color: rgb(153, 193, 241);
border:1px solid;
border-color: rgb(28, 113, 216);
border-radius:5px;

</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>20</y>
      <width>111</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Waga</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pora pomiaru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Poziom cukru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ciśnienie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Puls</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditWaga">
       <property name="styleSheet">
        <string notr="true">QLineEdit#lineEditWaga
{
background-color: rgb(19, 56, 91);
color: rgb(53, 132, 228);
border:1px solid;
border-color: rgb(28, 113, 216);
}
QLineEdit#lineEditWaga:focus
{
border:1px solid red;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxPora">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPoziomy">
       <property name="styleSheet">
        <string notr="true">QLineEdit#lineEditPoziomy
{
background-color: rgb(19, 56, 91);
color: rgb(53, 132, 228);
border:1px solid;
border-color: rgb(28, 113, 216);
}
QLineEdit#lineEditPoziomy:focus
{
border:1px solid red;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCisnienie">
       <property name="styleSheet">
        <string notr="true">QLineEdit#lineEditCisnienie
{
background-color: rgb(19, 56, 91);
color: rgb(53, 132, 228);
border:1px solid;
border-color: rgb(28, 113, 216);
}
QLineEdit#lineEditCisnienie:focus
{
border:1px solid red;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPuls">
       <property name="styleSheet">
        <string notr="true">QLineEdit#lineEditPuls
{
background-color: rgb(19, 56, 91);
color: rgb(53, 132, 228);
border:1px solid;
border-color: rgb(28, 113, 216);
}

QLineEdit#lineEditPuls:focus
{
border:1px solid red;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="zapiszWyniki">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Zapisz Wyniki</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wyczyscPola">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Wyczyść Pola</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Pokaż Ostatnie Wyniki</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="info">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FreeSerif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Okno informacyjne</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="etykieta">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>530</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Carlito</family>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="toolTip">
     <string>Program Cukrzyca w wersji 2.0 Open Source</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 56, 91);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string>Cukrzyca v.2 QT @ Slavo Heys&amp;2025</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUstawienia">
    <property name="title">
     <string>Ustawienia</string>
    </property>
    <addaction name="actionDanePersonalne"/>
    <addaction name="actionWydrukujWyniki"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionO_autorze"/>
    <addaction name="actioninfo"/>
    <addaction name="actionpomocna_d_o"/>
   </widget>
   <addaction name="menuUstawienia"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDanePersonalne">
   <property name="text">
    <string>dane personalne</string>
   </property>
  </action>
  <action name="actionWydrukujWyniki">
   <property name="text">
    <string>wydrukuj wyniki</string>
   </property>
  </action>
  <action name="actionO_autorze">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ContactNew"/>
   </property>
   <property name="text">
    <string>o autorze</string>
   </property>
  </action>
  <action name="actioninfo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DialogInformation"/>
   </property>
   <property name="text">
    <string>info</string>
   </property>
  </action>
  <action name="actionpomocna_d_o">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>pomocna dłoń</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
